GiniVision.podspec

```diff
@@ -15,11 +15,24 @@ The Gini Vision Library for iOS provides functionality to capture documents with
   s.author           = { 'Peter Pult' => 'p.pult@gini.net' }
   s.source           = { :git => 'https://github.com/gini/gini-vision-lib-ios.git', :tag => s.version.to_s }
   s.social_media_url = 'https://twitter.com/gini'
-
   s.ios.deployment_target = '8.0'
+  s.default_subspec = 'Core'
+
+  s.subspec 'Core' do |core|
+    core.source_files = 'GiniVision/Classes/Core/**/*'
```

Comment:
I honestly prefer the way they did it in the blog post. They added the core classes in the podspec so they were automatically added to all subspecs. The only downside I see is that the "Core" subspec would be empty.

Reply:
You are right, it would much better that way.